bin/fin

```diff
@@ -1431,7 +1431,9 @@ _vhost_proxy_connect ()
 	if [[ $? == 0 ]]; then
 		echo-green "Connected vhost-proxy to \"${network}\" network."
 		# Run a dummy container to trigger docker-gen to refresh proxy configuration.
-		docker run --rm --entrypoint=echo ${IMAGE_VHOST_PROXY} >/dev/null 2>&1
```

Comment:
This change is not necessary. It technically does not harm, but visually it adds complexity to the code.

Reply:
@lmakarov the function is ran when connecting containers to the vhost proxy so was thinking that it may be useful. But, if you don't think it is.